openreview/venue/group.py

```diff
@@ -340,62 +340,66 @@ def create_authors_group(self):
     def create_reviewers_group(self):
 
         venue_id = self.venue.id
-        reviewers_id = self.venue.get_reviewers_id()
-        area_chairs_id = self.venue.get_area_chairs_id()
-        senior_area_chairs_id = self.venue.get_senior_area_chairs_id()
-        reviewer_group = openreview.tools.get_group(self.client, reviewers_id)
-        if not reviewer_group:
-            reviewer_group = Group(id=reviewers_id,
-                            readers=[venue_id, senior_area_chairs_id, area_chairs_id, reviewers_id],
-                            writers=[venue_id],
-                            signatures=[venue_id],
-                            signatories=[venue_id],
-                            members=[]
-                        )
+        for index, role in enumerate(self.venue.reviewer_roles):
+
+            reviewers_id = self.venue.get_committee_id(role)
+            area_chairs_id = self.venue.get_committee_id(self.venue.area_chair_roles[index]) if index <= len(self.venue.area_chair_roles) else self.venue.get_area_chairs_id()
```

Comment:
I do not think we are validating anywhere that the arrays for the roles should be the same length. In this case, if the reviewer_roles array is longer than the area_chair_roles one, we will get an IndexError. Perhaps we should do if `index < len(self.venue.area_chair_roles)` so that if there are more reviewer roles, the last one gets the overall AC group as readers

Reply:
You're right! I forgot to mention that. Roles must be synced in all the arrays but I will make the change.

Reply:
Done.